layout/widgets/topplayers.php

```diff
@@ -1,24 +1,20 @@
-<div class="sidebar">
-	<h3>Top 5 players</h3>
-	<?php
-
-	$cache = new Cache('engine/cache/topPlayer');
-	if ($cache->hasExpired()) {
-		$players = mysql_select_multi('SELECT `name`, `level`, `experience` FROM `players` WHERE `group_id` < ' . $config['highscore']['ignoreGroupId'] . ' ORDER BY `experience` DESC LIMIT 5;');
-		
-		$cache->setContent($players);
-		$cache->save();
-	} else {
-		$players = $cache->load();
-	}
-
-	if ($players) {
-		$count = 1;
-		foreach($players as $player) {
-			echo "$count - <a href='characterprofile.php?name=". $player['name']. "'>". $player['name']. "</a> (". $player['level'] .").<br>";
-			$count++;
-		}
-	}
-	?>
-	<br>
-</div>
\ No newline at end of file
+    <div class="sidebar">
+        <h3>Top 5 players</h3>
+        <?php
+            $cache = new Cache('engine/cache/topPlayer');
+            if ($cache->hasExpired()) {
+                $players = mysql_select_multi('SELECT `name`, `level`, `experience`, `looktype`, `lookaddons`, `lookhead`, `lookbody`, `looklegs`, `lookfeet` FROM `players` WHERE `group_id` < ' . $config['highscore']['ignoreGroupId'] . ' ORDER BY `experience` DESC LIMIT 5;');
+                $cache->setContent($players);
+                $cache->save();
+            } else {
+                $players = $cache->load();
+            }
+            if ($players) {
+            $count = 1;
+            foreach($players as $player) {
+            echo '<img style="margin-top: -35px; margin-left: -35px;" src="https://outfit-images.ots.me/animatedOutfits1090/animoutfit.php?id='.$player['looktype'].'&addons='.$player['lookaddons'].'&head='.$player['lookhead'].'&body='.$player['lookbody'].'&legs='.$player['looklegs'].'&feet='.$player['lookfeet'].'&g=0&h=3&i=1"></img> <a href="characterprofile.php?name='.$player['name'].'">'.$player['name'].'</a>'. $player['level'].'<br>';
```

Comment:
Image server should be loaded from config.php:
https://github.com/Znote/ZnoteAAC/blob/c83f2d19faa1f12e7fdeaf367b4b7a957227ed29/config.php#L582